.github/FUNDING.yml

```diff
@@ -0,0 +1,8 @@
+# Core team members as per https://github.com/orgs/infection/teams/core/members
+github:
+  - sanmai
+  - localheinz
```

Comment:
Not sure about the number of sponsors, we probably need to verify that it allows adding this many members.

If it doesn't work, I think it would be best to focus on the core team members who are doing a lot of work - so feel free to take me off the list!

For reference, see https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/displaying-a-sponsor-button-in-your-repository.